Packs/RetarusSecureEmailGateway/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,6 @@
+
+#### Modeling Rules
+
+##### New: Retarus Secure Email Gateway Modeling Rule
+
+Added a new modeling rule for Retarus Secure Email Gateway MTA, MultiScan, PZD, CxO and Sandboxing events.
```

Comment:
Added a new data modeling rule for Retarus Secure Email Gateway MTA, MultiScan, PZD, CxO and Sandboxing events.